File: loex_Golang/cmd/accountwebsocketclientexample/accountwebsocketclientexample.go
package accountwebsocketclientexample

import (
	"fmt"
	"loex_golang/config"
	"loex_golang/logging/applogger"
	"loex_golang/pkg/client/accountwebsocketclient"
	"loex_golang/pkg/model/auth"
)

func RunAllExamples() {
	subAccountUpdate()
}

func subAccountUpdate() {
	// Initialize a new instance
	client := new(accountwebsocketclient.SubscribeAccountWebSocketV2Client).Init(config.AccessKey, config.SecretKey, config.Host)

	// Set the callback handlers
	client.SetHandler(
		// Authentication response handler
		func(resp *auth.WebSocketV2AuthenticationResponse) {
			if resp.IsSuccess() {
				client.Subscribe("")
			} else {
				applogger.Error("Authentication error, code: %d, message:%s", resp.Code, resp.Message)
			}
		},
		// Response handler
		func(resp interface{}) {
			applogger.Info("WebSocket Resp : %s", resp)
		})

	// Connect to the server and wait for the handler to handle the response
	client.Connect(true)

	fmt.Println("Press ENTER to unsubscribe and stop...")
	fmt.Scanln()

	// Unsubscribe the topic
	client.UnSubscribe("")

	// Close the connection
	client.Close()
	applogger.Info("Client closed")
}

File: loex_CSharp/Loex.SDK.Example/AccountWebSocketClientExample.cs
using System;
using Loex.SDK.Core.Client;
using Loex.SDK.Model.Response;
using Loex.SDK.Model.Response.Account;
using Loex.SDK.Model.Response.Auth;

namespace Loex.SDK.Example
{
    public class AccountWebSocketClientExample
    {
        public static void RunAll()
        {
            SubscribeAccount();
        }

        

        private static void SubscribeAccount()
        {
            // Initialize a new instance
            var client = new SubscribeAccountWebSocketClient(Config.AccessKey, Config.SecretKey, Config.Host);

            // Add the auth receive handler
            client.OnAuthenticationReceived += Client_OnAuthReceived;
            void Client_OnAuthReceived(WebSocketV2AuthResponse response)
            {
                if (response.code == (int)ResponseCode.Success)
                {
                    // Subscribe the specific topic
                    //client.Subscribe("1");
                    AppLogger.Info($"WebSocket authentication success, code={response.code}");
                }
                else
                {
                    AppLogger.Error($"WebSocket authentication fail, code={response.code}, message={response.message}");
                }
            }

            // Add the data receive handler
            client.OnDataReceived += Client_OnDataReceived;
            void Client_OnDataReceived(String response)
            {
                if (response != null)
                {
                    AppLogger.Info($"WebSocket returned data={response}");
                }
            }

            // Then connect to server and wait for the handler to handle the response
            client.Connect();

            client.Subscribe("111111111");


            Console.WriteLine("Press ENTER to unsubscribe and stop...\n");
            Console.ReadLine();

            // Unsubscrive the specific topic
            //client.UnSubscribe("1");

            // Delete handler
            client.OnDataReceived -= Client_OnDataReceived;
        }
    }
}
